client/src/components/Jobs/Jobs.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import fetchJobs from '../../hooks/fetchJobs.jsx';
+import JobListing from './JobListing.jsx';
+import JobDetails from './JobDetails.jsx';
+
+const Jobs = () => {
+  const [params, setParams] = useState({});
+  const [page, setPage] = useState(1);
+  const [currentJob, setCurrentJob] = useState(0);
+  const { jobs, loading, error } = fetchJobs(params, page);
+
+  return (
+    <div style={{ display: 'flex' }}>
+      {loading && <h1>Loading...</h1>}
+      {error && <h1>Error</h1>}
+      <div className='listings' style={{ flex: '50%' }}>
```

Comment:
I'd prefer not to use inline styles here and I'm not quite sure this is how flex is used. See https://css-tricks.com/snippets/css/a-guide-to-flexbox/#flex